### PR TITLE
Add back the cloudsigma-api feature for backwards compatibility

### PR DESCRIPTION
**Pull request by neykov**

### CODE REVIEW

**Comment by andreaturli** [comment]
Out of curiosity, jclouds-cli requires it?

File: feature/src/main/feature/feature.xml [hunk 1]
```diff
@@ -516,32 +521,27 @@ limitations under the License.
     </feature>
 
     <feature name='jclouds-cloudsigma2-hnl' description='CloudSigma v2 Honolulu' version='${project.version}'>
-        <feature version='${project.version}'>jclouds-compute</feature>
```
Comment (neykov) [inline feature/src/main/feature/feature.xml]: Goot point @demobox. I reverted the changes for the sake of consistency. Still think it's the right thing to do though. Perhaps something to take care of in another PR.

**Comment by neykov** [comment]
@andreaturli Either this PR or https://github.com/jclouds/jclouds-cli/pull/37 will fix the build. Thought it's useful to add anyway for backwards compatibility, in case other projects depend on it.

**Comment by demobox** [comment]
@neykov This looks good to me as long as we're confident that adding this back will not break the CLI in any way.

Out of curiosity: were you able to carry out your tests from jclouds/jclouds-cli#37 with this change added?

**Comment by demobox** [comment]
Er, actually: now that #98 is merged, does this need to be added to the new `jclouds-labs-all-compute` feature?

**Comment by neykov** [comment]
Good spot @demobox. Updated.
Now testing with jclouds-cli, though "it shouldn't matter (tm)" :).

**Comment by demobox** [comment]
> though "it shouldn't matter (tm)" :).

_grin_. Thanks for running the tests again!

**Comment by demobox** [comment]
[From Svet](https://github.com/jclouds/jclouds-cli/pull/37#issuecomment-287800676):

"Tested successfully..."

**Comment by demobox** [comment]
Merged to [master](https://git-wip-us.apache.org/repos/asf?p=jclouds-karaf.git;a=commit;h=7006c512427d70ba3c748f4e4a13c5f7546f4dd9). Thanks, Svet!

@neykov @andreaturli @nacx Does this or any of the related changes here or in jclouds-cli need backporting to any other branches, by the way?

**Comment by nacx** [comment]
I don't think so. The changes upgrade the version of karaf so we don't want them in the bugfix branches. Thanks for your help on this!

**Pull request closed without merging**